ReSharper.FSharp/test/data/features/quickFixes/replaceLambdaWithInnerExpression/Parens 02.fs

```diff
@@ -0,0 +1,3 @@
+let f x y = ()
+
+f (fun x ->{caret} f 1 x) 2
```

Comment:
Could you also add a similar test where parens are removed at the argument position, please?